src/nft/components/collection/CollectionNfts.tsx

```diff
@@ -66,9 +66,19 @@ const rarityStatusCache = new Map<string, boolean>()
 
 const ActionsContainer = styled.div`
   display: flex;
+  flex: 1 1 auto;
   gap: 10px;
-  width: 100%;
   justify-content: space-between;
+  margin: 0 16px;
+  @media screen and (min-width: ${({ theme }) => theme.breakpoint.sm}px) {
+    margin: 0 20px;
+  }
+  @media screen and (min-width: ${({ theme }) => theme.breakpoint.md}px) {
+    margin: 0 26px;
+  }
+  @media screen and (min-width: ${({ theme }) => theme.breakpoint.lg}px) {
+    margin: 0 48px;
+  }
```

Comment:
nit: I think the general pattern is to go from default to smaller screens with `max-width`. Looking at the codebase, we have 2x more occurrences of `max-width`!

Reply:
this is not particularly consistent enough to require this yet. i think once we decide we can add a lint rule